projects/packages/google-fonts-provider/src/introspectors/class-global-styles.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Google Fonts package Global Styles fonts introspector class file.
+ *
+ * @package automattic/jetpack-google-fonts-provider
+ */
+
+namespace Automattic\Jetpack\Fonts\Introspectors;
+
+use Automattic\Jetpack\Fonts\Utils;
+
+/**
+ * Global Styles fonts introspector.
+ */
+class Global_Styles {
+	/**
+	 * Enqueue fonts used in global styles settings.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_global_styles_fonts() {
+		if ( is_admin() || ! function_exists( 'wp_enqueue_webfont' ) ) {
```

Comment:
`Non-blocking`: This might be a silly question, but would you mind explaining the `is_admin()` condition here? I thought that we wanted to enqueue all fonts in the site editor? Or is that already being done elsewhere?

Reply:
Answering now for posterity (sorry I missed this initially!): this is because the all the fonts are already enqueued in wp-admin views (at least in the editor) by Gutenberg, so there's no need to run this function in that context.